pkg/client/unversioned/extensions.go

```diff
@@ -130,24 +130,24 @@ func NewExtensionsOrDie(c *Config) *ExtensionsClient {
 
 func setExtensionsDefaults(config *Config) error {
 	// if experimental group is not registered, return an error
-	g, err := latest.Group("extensions")
-	if err != nil {
-		return err
+	if !extlatest.IsEnabled() {
```

Comment:
If a user of our client library only cares about one group, then in the old code, the user can remove other groups from [import_known_versions.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/client/unversioned/import_known_versions.go), and those APIs won't be imported, and the client will compile. But with this PR, all groups will be imported.

This is an imaginary use case in my mind, I don't know if it makes sense in the real life.